notifications/README.md

```diff
@@ -0,0 +1,58 @@
+# Notifications
+
+## Notification Object
+
+```
+{
+  id: string,
+  username: string,
+  message: string,
+  createOn: unix-timestamp,
+  url: string,
+  type: string
+}
+```
+
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|      [GET /notifications](#get-notifications)      | Returns notifications of the logged in user |
+
+## **GET /notifications**
+
+Returns all notifications for a logged in user
+
+- **Params**  
+  None
+- **Query**  
+  page=[integer], n=[integer]
```

Comment:
What is `n`?

Reply:
number of items on the page (for pagination)

Reply:
Can we have that description in the docs please?

Reply:
Sure. Will do that.